modules/customize-certificates-add-service-serving.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-{context}"]
+= Add a service certificate
+
+To secure communication to your service, have the cluster generate a 
+signed serving certificate and key pair into a secret in your namespace. To 
```

Comment:
"your namespace" -> "same namespace as the service"

Reply:
Updated to use the recommended verbiage.